src/pages/integrations/bitpay-card/bitpay-card-topup/bitpay-card-topup.ts

```diff
@@ -515,10 +515,6 @@ export class BitPayCardTopUpPage {
                 );
                 let newAmountSat = maxAmountSat - invoiceFeeSat;
 
-                // Set expiration time for this invoice
```

Comment:
Why it's removed?

Reply:
That's the expiration time of the first invoice created that it's not needed

Reply:
Good!